docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<blockquote>This dataset lacks some of the aforementioned attributes, so we need to add them.</blockquote><p>On ne mentionne pas les attributs manquants dans ce notebook, donc il faudrait modifier ce commentaire un peu.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>